BUILDING.md

```diff
@@ -647,22 +647,25 @@ To test if Node.js was built correctly:
 > Release\node -e "console.log('Hello from Node.js', process.version)"
 ```
 
-### Android/Android-based devices (e.g. Firefox OS)
+### Android
 
 Android is not a supported platform. Patches to improve the Android build are
 welcome. There is no testing on Android in the current continuous integration
 environment. The participation of people dedicated and determined to improve
 Android building, testing, and support is encouraged.
 
 Be sure you have downloaded and extracted
-[Android NDK](https://developer.android.com/tools/sdk/ndk/index.html) before in
+[Android NDK](https://developer.android.com/ndk) before in
 a folder. Then run:
 
 ```console
-$ ./android-configure /path/to/your/android-ndk
-$ make
+$ ./android-configure /path/to/your/android-ndk target_arch android_sdk_version
```

Comment:
I tried using `./android-configure` to setup android build, but the shell complain that: 

```shell
return: can only `return' from a function or sourced script
```

I believe we should change `./android-configure` to `source ./android-configure` as the comment:

https://github.com/nodejs/node/blob/main/android-configure#L3-L4

Could you help to change to `source ./android-configure`?

Reply:
This seems interesting because I didn't seem to have this problem during actual testing, but I think you're right, and I'll fix it by committing a new commit after testing.

Reply:
Fixed.